en/micro-integrator/docs/references/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-example.md

```diff
@@ -0,0 +1,106 @@
+   <?xml version="1.0" encoding="UTF-8"?>
+   <sequence name="request" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+       <log level="full"/>
+   </sequence>
+   ```
+> **Note**: To configure the `secretKey` and `accessKey` parameter value, please use the [Setting up the Amazon Lambda Environment](../amazonlambda-connector/setting-up-amazonlambda.md) documentation.
+> - **secretKey** : The secret key used to sign requests.
+> - **accessKey** : The access key that corresponds to the secret key that you used to sign the request.
+> - **destination** : URL of the Amazon SQS Queue from which you want to consume messages.
+   
+## Exporting Integration Logic as a CApp
+
+**CApp (Carbon Application)** is the deployable artefact on the Enterprise Integrator runtime. Let us see how we can export integration logic we developed into a CApp. To export the `Solution Project` as a CApp, a `Composite Application Project` needs to be created. Usually, when a solution project is created, this project is automatically created by Integration Studio. If not, you can specifically create it by navigating to  **File** -> **New** -> **Other** -> **WSO2** -> **Distribution** -> **Composite Application Project**. 
+
+1. Right click on Composite Application Project and click on **Export Composite Application Project**.</br> 
+  <img src="../../../../assets/img/connectors/capp-project1.png" title="Export as a Carbon Application" width="300" alt="Export as a Carbon Application" />
+
+2. Select an **Export Destination** where you want to save the .car file. 
+
+3. In the next **Create a deployable CAR file** screen, select inbound endpoint and sequence artifacts and click **Finish**. The CApp will get created at the specified location provided in the previous step. 
+
+## Deployment
+
+1. Navigate to the [connector store](https://store.wso2.com/store/assets/esbconnector/list) and search for `AmazonSQS Connector`. Click on `AmazonSQS Inbound Endpoint` and download the .jar file by clicking on `Download Inbound Endpoint`. 
+   > **Note**: Copy this .jar file into **<PRODUCT-HOME>/dropins** folder.
+  
+2. Copy the exported carbon application to the **<PRODUCT-HOME>/repository/deployment/server/carbonapps** folder. 
+
+3. [Start the WSO2 EI server](https://ei.docs.wso2.com/en/latest/micro-integrator/overview/quick-start-guide/#start-the-micro-integrator). 
+
+## Testing  
+
+   Please log in to the Amazon **Simple Queue Service**-> created **Queue**. Select the Queue and **right click**-> **Send a Message**-> enter **Message**.
+   
+   **Sample Message**
+   
+   ```
+   {"Message":"Test Amazon SQS Service"}
+   ```
+   AmazonSQS Inbound Endpoint will consume message from the Simple Queue Service.
+   
+   **Expected respons**
+   
+   ```
+   [2020-05-22 12:28:03,799]  INFO {org.apache.synapse.mediators.builtin.LogMediator} - To: , MessageID: urn:uuid:CB783799949CD049281590130683750, Direction: request, Payload: {"Message":"Test Amazon SQS Service"}
+   ```
```

Comment:
You will see following message in WSO2 EI log file (found at <EI_HOME>/repository/logs/wso2carbon.log)

Reply:
Updated.